### PR TITLE
PLATFORM-2144 Add image-review translations

### PR DESCRIPTION
**Pull request by RafalWilinski**
## Links
* http://wikia-inc.atlassian.net/browse/PLATFORM-2144

## Description
This PR adds image-review strings translated to more languages.

## Reviewers
@vforge @Warkot

### CODE REVIEW

**Comment by RafalWilinski** [comment]
cc @hakubo @rogatty

File: crowdin/image-review.conf [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[crowdin.project]
+projectid = "image-review"
```
Comment (rogatty) [inline crowdin/image-review.conf]: I don't think it's right. Every other config file has `mercury` in here. If you want to understand why, @Warkot is the right person to ask.

**Comment by RafalWilinski** [comment]
Updated!

File: front/common/public/locales/pl/image-review.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "error-other":"Nie można załadować przeglądu obrazów",
+    "error-invalid-data":"Image Review API zwróciło błędne dane",
+    "error-no-more-images":"Nie ma więcej obrazów",
+    "review-and-get-more":"Sprawdź i przejdź do następnej partii",
+    "get-flagged":"Pobierz oflagowane",
+    "images-to-review-count":"Obrazy pozostałe do sprawdzenia:",
+    "image-context":"Kontekst:",
+    "image-context-not-provided":"Brak kontekstu",
+    "status-accepted":"OK",
+    "status-flagged":"Watpliwy",
```
Comment (rogatty) [inline front/common/public/locales/pl/image-review.json]: Should be "Wątpliwy" but I guess it's a mistake in Crowdin.

File: crowdin/image-review.conf [hunk 3]
```diff
@@ -0,0 +1,8 @@
+[crowdin.project]
+projectid = "mercury"
+source.base = "../front/common/public/locales"
```
Comment (rogatty) [inline crowdin/image-review.conf]: `repo.branch = "i18n-update"` is missing here comparing to other files. No idea what it's for ¯\_(ツ)_/¯

Comment (RafalWilinski) [inline crowdin/image-review.conf]: This branch no longer exists so I guess it's possible to skip that option and configure on-demand when doing any further changes to image review strings.

Comment (Warkot) [inline crowdin/image-review.conf]: This branch is not supposed to exist. This config option tells Crowdin CLI to create a brach with such name when using `-c` option. https://github.com/Wikia/i18n-tools#config
```bash
warkot@dev-warkot:~/mercury (dev)$ crowdin --project-config crowdin/main.conf download -c
[INFO]            Building project mercury...
[INFO]            Project mercury built with status: skipped
[INFO]            Downloading translations and extracting into /usr/wikia/crowdin/current/tmp/crowdin.mercury.warkot/extract directory
[INFO]            Checking out branch i18n-update
[INFO]            Moving files to the right place
[INFO]            Attempting to commit changed files
[INFO]            Staging changed files
[INFO]            Committing changes
warkot@dev-warkot:~/mercury (i18n-update)$
```

**Comment by bkoval** [comment]
Looks like all the i18n files are missing new line feed at the end of file :)

**Comment by RafalWilinski** [comment]
cc @hakubo @rogatty

**Comment by rogatty** [comment]
LGTM

**Pull request merged**